libsnowflakeclient/deps-build/win64/vs14/azure/include/http/libcurl_http_client.h

```diff
@@ -327,8 +327,11 @@ namespace azure {  namespace storage_lite {
                 {
                   curl_easy_setopt(h, CURLOPT_PROXY, proxy_host.c_str());
                   curl_easy_setopt(h, CURLOPT_PROXYPORT, (long) proxy_port);
-                  curl_easy_setopt(h, CURLOPT_PROXYUSERNAME, proxy_user.c_str());
-                  curl_easy_setopt(h, CURLOPT_PROXYPASSWORD, proxy_password.c_str());
+                  if (!proxy_user.empty() || !proxy_password.empty())
+                  {
+                    curl_easy_setopt(h, CURLOPT_PROXYUSERNAME, proxy_user.c_str());
```

Comment:
same questions here, about the intended behavior when one of the values is empty.